index.html

```diff
@@ -55,6 +55,66 @@ <h2>Filtres</h2>
           cette ville
         </p>
       </section>
+      <!--section : the most popular-->
+      <section>
+        <div id="most_popular">
+          <!--mp heading-->
+          <div id="most_popular_heading">
+            <h2>Les plus populaires</h2>
+            <i class="fas fa-chart-line"></i>
+          </div>
+          <!--mp hotel 1-->
+          <a href="#" class="hotel_card">
+            <img
+              class="hotel_photo"
+              src="./assets/img/emile-guillemot-Bj_rcSC5XfE-unsplash.jpg"
+              alt="chambre hôtel Le soleil du matin"
+            />
+            <div class="hotel_info">
+              <h3>Hôtel Le soleil du matin</h3>
+              <p>Nuit à partir de 128<strong>€</strong></p>
+              <div class="hotel_score">
+                <i class="fas fa-star"></i><i class="fas fa-star"></i
+                ><i class="fas fa-star"></i><i class="fas fa-star"></i
+                ><i class="fas fa-star"></i>
+              </div>
```

Comment:
Use ul with five li for every star and add class "active" on on sould be blue

Reply:
ok done; do I leave or remove the hotel_score class ?